CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Fri Mar 22 2019 Joseph Sharkey <shark.bruhaha@gmail.com> - 3.15.0-0
+- Marked bracketize puppet 3 function as deprecated.
+- Created bracketize function, and spec test
```

Comment:
This comment appears to be incorrect.  I don't see any namespaced 4.x bracketize functions in this PR.